tbx/static_src/sass/components/_rich-text.scss

```diff
@@ -1,11 +1,42 @@
 @use "config" as *;
 
 .rich-text {
+    @include font-size(body);
+
+    h3 {
+        @include font-size(heading-three);
+        margin-bottom: $spacer-small;
+    }
+
+    h4 {
+        @include font-size(heading-four);
+        margin-bottom: $spacer-small;
+    }
+
     ul {
         list-style: inside disc;
+        margin-bottom: $spacer-small;
     }
 
     ol {
         list-style: inside decimal;
+        margin-bottom: $spacer-small;
+    }
+
+    a {
+        @include link-styles(var(--color--text), var(--color--text));
```

Comment:
Links are white globally but in rich text they are the same colour as the text so need to pass those values in here.